### ISSUE TITLE
Windows 10 throws warning at install

### ISSUE THREAD

**Issue by ZoeBijl** [ThePacielloGroup/aviewer#18]
Windows protected your PC
Windows SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.

App: aViewer2.0.exe 
Publisher:† Unknown publisher

**Comment by matatk** [comment]
I expect this is because the software is not signed (and will probably apply to WAT and CCA too). Not sure about the process/costs for getting it signed, but until then, we should probably warn users to expect this, via the documentation.

**Comment by patrickhlauke** [comment]
As our users are fairly tech-savvy (to an extent anyway), I think this is less of a concern (then again, I'm an optimist)

**Comment by matatk** [comment]
@patrickhlauke agreed; as long as it's in the docs - i.e. definitely in the README, but maybe also a standard part of release notes, if that can be automated, or perhaps on the website (though maybe that would look a bit negative), then I suspect that would be OK for now.

Some thing that just occurred to me:
1. @MichielBijl you say the error appeared at install time, but it seems from the text like aViewer attempted to start -- is this behaviour by design from the installer?
2. I wonder if there's a way for us to at least fill in the Publisher field in the .exe - I'd expect so.

**Comment by WW3** [comment]
same here, this message appears at install start time

**Comment by patrickhlauke** [comment]
the only way around the warning is to properly sign the excutable(s) (see for instance http://stackoverflow.com/questions/252226/signing-a-windows-exe-file), which requires purchase of a certificate...not a cost we've budgeted for.

**Comment by ZoeBijl** [comment]
You can get free certs for open source projects: https://www.certum.eu/certum/cert,offer_en_open_source_cs.xml

@matatk: this message was when I started the installation exe (also called aviewer.exe)

**Comment by patrickhlauke** [comment]
hmmm, €14.00 and a lot of paperwork still involved. not high on our priority list, but something @stevefaulkner and i will raise
